tests/integration/test_config.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd move this file to ``tests/integrations/config/views/test_suggesters.py`` WDYT?